test/jdk/sun/util/resources/TimeZone/Bug4848242.java

```diff
@@ -23,9 +23,12 @@
 
 /*
  *@test
- *@bug 4848242
- *@summary Make sure that MET time zone is not misinterpreted in euro locales.
- *         Display the MET and MEST TZ human-readable name in all euro locales.
+ *@bug 8342886
```

Comment:
The bug id needs to be appended

```suggestion
 *@bug 4848242 8342886
```